helpers/addresses.py

```diff
@@ -425,6 +425,7 @@
         "restitution": "0x3F61344BA56df00dad9bBcA05d98CA2AeC43Ba0B",
         "bcrvBadger": "0xa54d8a596B4022CC4436b692C8ea0E342405eB6e",
         "dummy": "0xEC4bc89F35A7bf9ffe0375Fd67940Ded1e2253BF",
+        "minting": "0xebA06639750BADcDf5EE100D72F3C93f2D1918A7",
```

Comment:
where did you get this address from?
```
function generateTokens(address _owner, uint _amount) onlyController public returns (bool) {
```
```
modifier onlyController {
    require(msg.sender == controller);
    _;
}
```
```
>>> badger = Contract('0x3472A5A71965499acd81997a54BBA8D852C6E53d')
>>> badger.controller()
'0xdDB2dfad74F64F14bb1A1cbaB9C03bc0eed74493'
```

Reply:
somehow i ended up using the implementation of the controller proxy, instead of the proxy itself, good catch

resolved in 703863a55f39bc59f8f5090b208d55a50b576944